app/app.go

```diff
@@ -118,15 +61,15 @@ func New(options ...Option) (outApp *App, outErr error) {
 	rootRouter := mux.NewRouter()
 
 	app := &App{
-		goroutineExitSignal: make(chan struct{}, 1),
 		Srv: &Server{
```

Comment:
Unrelated to this change, but longterm would it make sense to `s/Srv/Server`?

Reply:
I may or may not remove the server entirely from the App at some point. (having interfaces to the various parts, depends on if I feel it necessary)